Makefile

```diff
@@ -81,6 +81,9 @@ helm: ## Generate Helm Chart
 website: ## Generate Docs Website
 	cd website;npm install;hugo -d ../docs
 
+website: ## Generate Docs Website
+	cd website; npm install; git submodule update --init --recursive; hugo
+
```

Comment:
is this duplicating the same action?

Reply:
This is, a bad merge resolution artifact. Removing now